staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -922,3 +818,67 @@ func TestFinishRequest(t *testing.T) {
 		}
 	}
 }
+
+func (tc *patchTestCase) setTcStartingPod(name string, namespace string, uid types.UID, resourceVersion string, apiVersion string, activeDeadlineSeconds *int64, nodeName string) {
+	tc.startingPod.Name = name
```

Comment:
all 4 funcs are the same, with a different pod. Please factor the common code out.

Reply:
Done, Thanks for your review.